### DISCUSSION TITLE
How to use jwt with rtpm

### DISCUSSION THREAD

**Discussion by irg1008** [bluenviron/mediamtx#3491] [Questions]
### Question

Hi, sorry if this a fairly straight forward question but How can I pass the jwt token via rtmp for authentication. I tried using jwt query param like rtpm://localhost with streamKey and jtw query param

**Comment by aler9** [comment]
Hello, the README says

> Clients are expected to pass the JWT in the Authorization header (in case of HLS and WebRTC) or in query parameters (in case of all other protocols)

Therefore:

```
rtmp://localhost/mypath?jwt=MY_JWT
```

[a comment by an automated account is not shown]
